MIGRATION_GUIDE.md

```diff
@@ -374,17 +374,31 @@ Implementing the converter changes from Morphia interface to using the Spring in
+|-----------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `@Entity(noClassnameStored = true, name="exampleEntity")` | `@Document("exampleEntity")`. There is no property similar to _noClassnameStored_ as the type/class can't be excluded with Spring Data.                                                                                                     |
+| `@PrePersist`                                             | There is no replacement annotation for this. If you are using this on any fields, please set the field before save(). One very common example is to set the creation date like this `entity.setCreated(ZonedDateTime.now(ZoneOffset.UTC));` |
+| `@Embedded`                                               | No replacement available as Spring Data already embeds the document.                                                                                                                                                                        |
+| `@Converters()`                                           | Replaced with `@ReadingConverter` and `@WritingConverter`                                                                                                                                                                                   |
 
 
 ## 5 sda-commons-server-kafka
 
 Some deprecated code was removed.
 Especially we removed the feature `createTopicIfMissing` because we don't recommend services
 to do that. You usually need different privileges for topic creation that you don't want to
-give to your services.
\ No newline at end of file
+give to your services.
+
+We also removed the topicana classes in package `org.sdase.commons.server.kafka.topicana`. For this
+reason the following methods were removed
+on [MessageListenerRegistration](./sda-commons-server-kafka/src/main/java/org/sdase/commons/server/kafka/builder/MessageListenerRegistration.java)
+and [ProducerRegistration](./sda-commons-server-kafka/src/main/java/org/sdase/commons/server/kafka/builder/ProducerRegistration.java):
+
+### checkTopicConfiguration
+This method compared the current topics with the configured ones and threw a `org.sdase.commons.server.kafka.topicana.MismatchedTopicConfigException` in case they did not match.
+
+### forTopicConfigs
+These methods accepted a list of `org.sdase.commons.server.kafka.topicana.ExpectedTopicConfiguration` to compare against the current topic configuration.
+
+## forTopic(ExpectedTopicConfiguration topic)
```

Comment:
Last header should probably use three #, i.e. "###"? You could also make them bullet points.